src/App.js

```diff
@@ -1,23 +1,27 @@
-import logo from './logo.svg';
+import { BrowserRouter, Route, Switch } from 'react-router-dom';
+import Home from './Views/Home/Home';
+import CustomHook from './Components/CustomHook/CustomHook';
+import Footer from './Views/Footer/Footer';
 import './App.css';
+import EntryList from './Components/EntryList/EntryList';
 
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <div className="navbar">
+        <CustomHook />
```

Comment:
In the future I'd recommend naming this something more explicit, like DarkModeTheme or something similar.